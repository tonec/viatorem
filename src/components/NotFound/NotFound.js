import React from 'react'
import { OneColumnThin } from 'components'

export default function NotFound () {
  return (
    <OneColumnThin title="404 page not found">
      <h1>Doh! 404!</h1>
      <p>
        These are <em>not</em> the droids you are looking for!
      </p>
    </OneColumnThin>
  )
}
