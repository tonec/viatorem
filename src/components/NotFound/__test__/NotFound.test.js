import React from 'react'
import { shallow } from 'enzyme'
import NotFound from '../NotFound'

describe('NotFound (snapshot)', () => {
  it('renders according to snapshot', () => {
    const component = shallow(<NotFound />)
    expect(component).toMatchSnapshot()
  })
})
