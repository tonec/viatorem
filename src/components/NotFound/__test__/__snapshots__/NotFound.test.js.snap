// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound (snapshot) renders according to snapshot 1`] = `
<OneColumnThin
  className=""
  offset={9}
  span={6}
  title="404 page not found"
>
  <h1>
    Doh! 404!
  </h1>
  <p>
    These are 
    <em>
      not
    </em>
     the droids you are looking for!
  </p>
</OneColumnThin>
`;
